shared/common-adapters/icon.constants-gen.tsx

```diff
@@ -6795,13 +6803,41 @@ const iconMeta_ = {
       return require('../images/icons/icon-play-64.png')
     },
   },
+  'icon-poop-32': {
```

Comment:
no dark-poop for 32 thru 64?

Reply:
It hadn't been exported on zeplin but it is now, so I'll add it.